instrumentation/akka/akka-http-10.0/javaagent/src/test/scala/io/opentelemetry/javaagent/instrumentation/akkahttp/AbstractHttpServerInstrumentationTest.scala

```diff
@@ -42,5 +42,8 @@ abstract class AbstractHttpServerInstrumentationTest
     )
     // instrumentation does not create a span at all
     options.disableTestNonStandardHttpMethod
+    options.setExpectedException(
```

Comment:
I'd just make the scala and grails tests also throw `IllegalStateException`. The only one tests that could keep using are the old groovy tests, they'll get converted eventually anyway.